notebooks/dynamicrafter-animating-images/dynamicrafter-animating-images.ipynb

```diff
@@ -0,0 +1,696 @@
+{
```

Comment:
<p>it is better to explain which models also available (if user wants convert and run different model manually) and that this model selected as example, steps for other models will be similar</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Extra information and link to models were added</p>